fmralign/alignment_methods.py

```diff
@@ -369,6 +374,7 @@ def fit(self, X, Y):
         Y: (n_samples, n_features) nd array
             target data
         """
+        import ot
```

Comment:
Why are we moving this inside the function ? It's included in our base dependencies, so it's assumed installed